src/test/java/de/tracetronic/jenkins/plugins/ecutest/tool/installation/ConfigurationAsCodeIT.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 2015-2019 TraceTronic GmbH
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+package de.tracetronic.jenkins.plugins.ecutest.tool.installation;
+
+import de.tracetronic.jenkins.plugins.ecutest.IntegrationTestBase;
+import de.tracetronic.jenkins.plugins.ecutest.wrapper.com.ETComProperty;
+import io.jenkins.plugins.casc.ConfigurationAsCode;
+import io.jenkins.plugins.casc.snakeyaml.Yaml;
+import org.junit.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.net.URL;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import static org.hamcrest.Matchers.empty;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThat;
+
+/**
+ * Integration tests for {@link ConfigurationAsCode} compatibility.
+ *
+ * @author Christian Pönisch <christian.poenisch@tracetronic.de>
+ */
+public class ConfigurationAsCodeIT extends IntegrationTestBase {
+
+    @Test
+    public void testImportConfiguration() throws Exception {
+        final URL yamlConfig = this.getClass().getResource("configuration-as-code.yml");
+        System.out.println(yamlConfig);
+        ConfigurationAsCode.get().configure(yamlConfig.toString());
```

Comment:
Consider using the annotation `@ConfiguredWithCode("configuration-as-code.yml")` which the `tests` classifier includes.